CHANGELOG.md

```diff
@@ -7,205 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Add `pasteIntoField` webdriver method ([#14004](https://github.com/MetaMask/metamask-extension/pull/14004))
-- Dark Mode: Fix colors in threedot asset menu ([#13990](https://github.com/MetaMask/metamask-extension/pull/13990))
-- Dark Mode:  Depost modal ([#13989](https://github.com/MetaMask/metamask-extension/pull/13989))
-- Dark Mode: Update login screeen colors ([#13992](https://github.com/MetaMask/metamask-extension/pull/13992))
-- E2e Swap Api mockup response for suggestedGasFees ([#13991](https://github.com/MetaMask/metamask-extension/pull/13991))
-- Make Edit Gas Fee testcase independent of arbitrary Wait time for Gas Editing ([#13888](https://github.com/MetaMask/metamask-extension/pull/13888))
-- General backgrounds and borders design token updates ([#13764](https://github.com/MetaMask/metamask-extension/pull/13764))
-- Add `IconEye` and `IconEyeSlash` icon components ([#13978](https://github.com/MetaMask/metamask-extension/pull/13978))
-- Bump controllers to 27 ([#13985](https://github.com/MetaMask/metamask-extension/pull/13985))
-- Add `data-test-id` prop to `TextField` component ([#13973](https://github.com/MetaMask/metamask-extension/pull/13973))
-- Dark Mode: Select Account Hardware Wallets ([#13917](https://github.com/MetaMask/metamask-extension/pull/13917))
-- Update package.json ([#13961](https://github.com/MetaMask/metamask-extension/pull/13961))
-- E2e metrics ([#13904](https://github.com/MetaMask/metamask-extension/pull/13904))
+- Improving settings toggle accessibility by allowing label interaction ([#13876](https://github.com/MetaMask/metamask-extension/pull/13876))
 - Updating account and network icons ([#13947](https://github.com/MetaMask/metamask-extension/pull/13947))
-- Dark Mode: Update transaction detail activity log ([#13919](https://github.com/MetaMask/metamask-extension/pull/13919))
-- Automate the Flask release process ([#13898](https://github.com/MetaMask/metamask-extension/pull/13898))
-- Add labels to provide better accessibility during onboarding v2 ([#13890](https://github.com/MetaMask/metamask-extension/pull/13890))
-- Dark Mode: Fix currently selected network color ([#13918](https://github.com/MetaMask/metamask-extension/pull/13918))
-- Fix yarn.lock ([#13941](https://github.com/MetaMask/metamask-extension/pull/13941))
-- Permissions: convert hook to util ([#13906](https://github.com/MetaMask/metamask-extension/pull/13906))
-- Remove usages of useMetricEvent hook ([#13930](https://github.com/MetaMask/metamask-extension/pull/13930))
-- Show snap version on install ([#13931](https://github.com/MetaMask/metamask-extension/pull/13931))
+
+### Fixed
+- Add 'Enhanced Gas UI' setting to search index ([#14206](https://github.com/MetaMask/metamask-extension/pull/14206))
+- Fixed issue where the "lock screen" shows up briefly between the "create password" screen and the success screen ([#14111](https://github.com/MetaMask/metamask-extension/pull/14111))
```

Comment:
`Fixed issue where the "lock screen" shows up briefly between the "create password" screen and the success screen` is behind a feature toggle and shouldn't be included